tests/code_wars/check_same_case_test.py

```diff
@@ -17,14 +17,13 @@
 def test_case():
```

Comment:
by convention the name of this file matches the name of the file being tested with the addition of a prefix or suffix `test`, _and_ the name of the function should also match the name of the method being tested with a prefix `test_` and perhaps a suffix describing what about the function is being tested if more than one test tests the same function.